quickstarts/Streaming.ipynb

```diff
@@ -43,16 +43,29 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        if chunk.text:
</pre><p>There are a few ways this can go wrong, and this only covers one of them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>